freeipa/src/main/java/com/sequenceiq/freeipa/service/stack/FreeIpaUpgradeCcmService.java

```diff
@@ -79,7 +79,7 @@ private OperationStatus alreadyFinishedOperationStatus(String environmentCrn) {
     }
 
     private void validateCcmUpgrade(Stack stack) {
-        if (!stack.isAvailable()) {
+        if (!stack.getStackStatus().getStatus().isCcmUpgradeablePhase()) {
             throw new BadRequestException(
                     String.format("FreeIPA stack '%s' must be AVAILABLE to start Cluster Connectivity Manager upgrade.", stack.getName()));
```

Comment:
`must be AVAILABLE` like above, this is not strictly true, but I do not mind leaving it as is.

Reply:
I did not want to display the "or if it is in CCM upgrade failed state"